src/reducers/projects.js

```diff
@@ -40,6 +49,8 @@ const tasks = (state, action) => {
       ])
     case START_TASK:
       return calculateCounters(state.list.map(t => task(t, action)))
+    case COMPLETE_TASK:
+      return calculateCounters(state.list.map(t => task(t, action)))
```

Comment:
looks like the code for both `START_TASK` and `COMPLETE_TASK` are the same so the cases could be collapsed